src/components/Calender.vue

```diff
@@ -97,44 +106,47 @@
 		transition: background-color 1.6s ease-in;
 	}
 
-	a.date.padding
-	{
+	a.date.padding {
 		opacity: 0;
 		pointer-events: none;
 	}
 
 	a.date:hover,
-	a.date:focus
-	{
+	a.date:focus {
 		background-color: var(--red);
 		transition: background-color 0.3s ease;
 	}
-	
-	h3
-	{
+
+	h3 {
 		font-family: var(--numbers-font);
 		font-size: 2.5rem;
 		font-weight: normal;
 
-		padding: 0.22em 0.11em 0;
 		margin: 0;
 		margin-bottom: auto;
-		border-radius: 100%;
+		text-align: left;
+	}
 
+	.number {
```

Comment:
A more descriptive name might be "date", but that's already in use on the page. I would think of the previous structure as _"a date, with the heading <insert number>"_, which makes sense to me because "26" is the title of the 26th, so to speak. I think the HTML structure and/or class naming needs to be adjusted here for this to still make sense.